internal/cluster/cluster.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"strings"
 
+	build2 "github.com/rh-ecosystem-edge/kernel-module-management/internal/utils/build"
```

Comment:
Why is this named `build2`? Why not just using `build`?

Reply:
Fixed, thanks!